app/frontend/shared/utils/StringUtils.res

```diff
@@ -94,7 +94,7 @@ let toColor = t => {
 let lengthBetween = (~allowBlank=true, t, min, max) => {
   let tLen = t->String.trim->String.length
   if tLen > 0 {
-    min <= tLen || tLen >= max
+    min <= tLen && tLen <= max
```

Comment:
This changes the behavior of `lengthBetween`, we are using it at other places, I think this will break the logic at places where it was used before; (why this change tho 🤔)